skll/utilities/filter_features.py

```diff
@@ -31,43 +31,50 @@ def main(argv=None):
+                        help='output feature file (must have same extension as '
+                             'input file)')
     parser.add_argument('-f', '--feature',
-                        help='A feature in the feature file you would like to \
-                              keep.  If unspecified, no features are removed.',
+                        help='A feature in the feature file you would like to '
+                             'keep.  If unspecified, no features are removed.',
                         nargs='*')
     parser.add_argument('-I', '--id',
-                        help='An instance ID in the feature file you would \
-                              like to keep.  If unspecified, no instances are \
-                              removed based on their IDs.',
+                        help='An instance ID in the feature file you would '
+                             'like to keep.  If unspecified, no instances are '
+                             'removed based on their IDs.',
                         nargs='*')
     parser.add_argument('--id_col',
-                        help='Name of the column which contains the instance \
-                              IDs in ARFF, CSV, or TSV files.',
+                        help='Name of the column which contains the instance '
+                             'IDs in ARFF, CSV, or TSV files.',
                         default='id')
     parser.add_argument('-i', '--inverse',
-                        help='Instead of keeping features and/or examples in \
-                              lists, remove them.',
+                        help='Instead of keeping features and/or examples in '
+                             'lists, remove them.',
                         action='store_true')
     parser.add_argument('-L', '--label',
-                        help='A label in the feature file you would like to \
-                              keep.  If unspecified, no instances are removed \
-                              based on their labels.',
+                        help='A label in the feature file you would like to '
+                             'keep.  If unspecified, no instances are removed '
+                             'based on their labels.',
                         nargs='*')
     parser.add_argument('-l', '--label_col',
-                        help='Name of the column which contains the class \
-                              labels in ARFF, CSV, or TSV files. For ARFF \
-                              files, this must be the final column to count as\
-                              the label.',
+                        help='Name of the column which contains the class '
+                             'labels in ARFF, CSV, or TSV files. For ARFF '
+                             'files, this must be the final column to count as '
+                             'the label.',
                         default='y')
+    parser.add_argument('-r', '--replace_blanks_with',
```

Comment:
```suggestion
    parser.add_argument('-rb', '--replace_blanks_with',
```

I don't like the `-r`. There's no reason that these have to be single letters so why not pick something a bit more intuitive? Of course, `-r` will still work since there's nothing else that conflicts but at least the documentation will display the more intuitive option.